### ISSUE TITLE
Error eMail Class

### ISSUE THREAD

**Issue by PowerPan** [Admidio/admidio#43]
I Get this Error based on there Error log

```
PHP Parse error:  syntax error, unexpected '}' in XXX/adm_program/system/classes/email.php on line 418, referer: XXX/adm_program/modules/profile/profile_new.php?new_user=1&lastname=Johannes&firstname=Rudolph
```

i see no error in the phpfile ^^

**Comment by sirnone** [comment]
http://forum.admidio.org/viewtopic.php?f=3&t=6119

in the file adm_program/system/classes/email.php - line 417 we missed a ";"

**Comment by PowerPan** [comment]
ok i see it is already fixed in the master branch ^^

**Comment by sirnone** [comment]
yes, but master branch will need a long beta-testing...
@Fasse: should we make an last Version 2.4 to fix this error?

**Comment by PowerPan** [comment]
I Think you should do this, without this fix you cannot save anything^^

**Comment by Fasse** [comment]
We should release a new version 2.4.12 with this bugfix. it was my mistake I didn't test my changes :(

**Comment by ximex** [comment]
isn't this fixed yet?

**Comment by sirnone** [comment]
fixed in 2.4.12
